packages/bezier-react/src/components/TextField/TextField.test.tsx

```diff
@@ -296,5 +296,25 @@ describe('TextField', () => {
       )
       expect(clearButton).toBeVisible()
     })
+
+    it('should focus the clear button when Tab key is pressed', async () => {
+      const { getByTestId } = renderComponent({
+        value: 'test',
+        allowClear: true,
+      })
+      const rendered = getByTestId(TEXT_INPUT_TEST_ID)
+      const input = rendered.getElementsByTagName('input')[0]
+
+      input.focus()
+      await user.tab()
+
+      const focusedElement = document.activeElement
+      expect(focusedElement).toHaveClass('CloseIconWrapper')
```

Comment:
```suggestion
      expect(document.activeElement).toHaveClass('CloseIconWrapper')
```

(마이너) 인라인으로 충분할 것 같습니다!

Reply:
수정 완료했습니다!